beets/autotag/mb.py

```diff
@@ -416,18 +416,17 @@ def album_info(release):
         if reltype:
             info.albumtype = reltype.lower()
 
-    # Log the new-style "primary" and "secondary" release types.
-    # Eventually, we'd like to actually store this data, but we just log
-    # it for now to help understand the differences.
+    # Set the new-style "primary" and "secondary" release types.
+    albumtypes = []
     if 'primary-type' in release['release-group']:
         rel_primarytype = release['release-group']['primary-type']
         if rel_primarytype:
-            log.debug('primary MB release type: ' + rel_primarytype.lower())
+            albumtypes.append(rel_primarytype.lower())
     if 'secondary-type-list' in release['release-group']:
         if release['release-group']['secondary-type-list']:
-            log.debug('secondary MB release type(s): ' + ', '.join(
-                [secondarytype.lower() for secondarytype in
-                    release['release-group']['secondary-type-list']]))
+            for sec_type in release['release-group']['secondary-type-list']:
+                albumtypes.append(sec_type.lower())
+    info.albumtypes = ','.join(albumtypes)
```

Comment:
It occurs to me somewhat belatedly, after reviewing #4044, that we've been using an existing convention for storing list-like data in string fields: namely, separating values with `; ` (semicolon-space) as opposed to just `,` (comma). I think this makes for somewhat more legible strings for humans and also is slightly less likely to get confused (because `,` seems more likely to occur within an individual element). Can we do that here?

Reply:
No problem; done!